interface/wx/cpp.h

```diff
@@ -52,9 +52,20 @@
 
     Please use the standard macro instead.
 
+    @see __WXFUNCTION_SIG__
+
     @header{wx/cpp.h}
 */
 #define __WXFUNCTION__ __func__
 
+/**
+    Display the current function's full signature, if available.
```

Comment:
I think it would be more correct to say
```suggestion
    Expands to the current function's full signature, if available.
```

Reply:
Fixed